src/common/packager.ts

```diff
@@ -43,7 +45,7 @@ export class Packager {
                             let args = ["--port", Packager.PORT];
                             let childEnvForDebugging = Object.assign({}, process.env, { REACT_DEBUGGER: "echo A debugger is not needed: " });
 
-                            Log.logMessage("Starting Packager", outputChannel);
+                            Log.logMessage("Starting Packager");
```

Comment:
At the entry point, we set the global logger for the extension process. If not, we default to the Console logger. In the past, we were erroneously creating multiple output channels with the same name and logging on it. This way, we will create the OutputChannel just once and use it.